PennMobile/Preferences/PreferencesView.swift

```diff
@@ -35,6 +35,7 @@ struct PreferencesView: View {
                     Text("Choose which features appear in the tab bar. The remaining ones can still be found on the More tab.")
                 }
             }
+            .id(UUID())
```

Comment:
What bug was this fixing?

Reply:
It fixes this weird visual glitch in the list when you click and hold for too long on one element
https://github.com/pennlabs/penn-mobile-ios/assets/35184414/e633e329-8501-4808-a155-da3f552530ee